pkg/appmanager/as3Manager_test.go

```diff
@@ -277,5 +277,19 @@ var _ = Describe("As3Manager Tests", func() {
 			Expect(reflect.DeepEqual(origCfg, generatedCfg)).To(BeTrue(), "Failed to Create JSON with correct configuration")
 		})
 	})
+	Describe("postRouteDeclarationHost", func() {
```

Comment:
Let's keep this change in the appropriate PR

Reply:
Sure @ssurenr will take the test case in an appropriate PR.